src/main/java/com/faforever/client/mod/ModManagerController.java

```diff
@@ -86,6 +97,22 @@ public void initialize() {
     loadActivatedMods();
 
     modListView.scrollTo(modListView.getSelectionModel().getSelectedItem());
+
+    modSearchTextField.textProperty().addListener(new ChangeListener<String>() {
+      @Override
+      public void changed(ObservableValue<? extends String> observable, String oldValue, String newValue) {
+        if (newValue.isEmpty()) {
+          modVersionFilteredList.setPredicate(null);
+        } else {
+          modVersionFilteredList.setPredicate(modVersion -> modVersion.getDisplayName().toLowerCase().contains(newValue.toLowerCase()));
+        }
+        if (!modVersionFilteredList.isEmpty()) {
+          modListView.getSelectionModel().select(0);
```

Comment:
Selection is used in a different way for the mod list. It indicates which mods are enabled so we likely do not want to be auto selecting mods for the user as this can get confusing especially since the list is multi select

Reply:
removed